### ISSUE TITLE
As someone who just watched a pull request get merged, I'd like to know how to retry the API test suite when an EC2 instance fails to launch

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#6291]
https://jenkins.dataverse.org/job/IQSS-dataverse-develop/217/console is a case in point. The failure was...

`ssh: connect to host ec2-54-80-11-163.compute-1.amazonaws.com port 22: Connection timed out`

... as shown in the screenshot below. This means that we don't know if the API test suite is passing or not. We should document in the Dev Guide what to do in this case. Sometimes with Travis I click the "restart" button. I don't know what to do with Jenkins (and I don't think I have a login, actually).

![Screen Shot 2019-10-18 at 11 04 43 AM](https://user-images.githubusercontent.com/21006/67105779-6518f180-f197-11e9-9d10-3071757a00b4.png)

The best entry point to these jobs is clicking the "API Test Status: failing" button in the README.

![Screen Shot 2019-10-18 at 11 07 24 AM](https://user-images.githubusercontent.com/21006/67105856-942f6300-f197-11e9-9251-07c223c7baa2.png)

**Comment by donsizemore** [comment]
@pdurbin i've bumped the wait time from 30 to 60 to 120 seconds (and still see a fair number of these)

**Comment by pdurbin** [comment]
@donsizemore thanks! Does the concept of a "Restart build" button even make sense in a Jenkins world? Here's how it looks in Travis:

![Screen Shot 2019-10-18 at 11 36 54 AM](https://user-images.githubusercontent.com/21006/67108078-b4f9b780-f19b-11e9-80b1-ab08cbe7681a.png)

**Comment by donsizemore** [comment]
@pdurbin Jenkins includes options to build on a new PR or an updated PR.

**Comment by pdurbin** [comment]
@donsizemore ok. In this case I'm talking about the "develop" branch. The idea with https://jenkins.dataverse.org/job/IQSS-dataverse-develop/ (please correct me if I'm wrong) is that we are catching regressions before we ship. The pull request has already been merged. Basically, that job is a replacement for the phoenix server, in my mind. It's step one of "thoughts on automated testing" at https://groups.google.com/d/msg/dataverse-dev/ISot5k4VjZQ/t-hzPk8tAwAJ which is this:

- Get https://jenkins.dataverse.org to do the same job that the older Jenkins server and phoenix do which is to catch regressions before a release.

That's the way I've been talking about this at standup anyway, that once we get https://jenkins.dataverse.org/job/IQSS-dataverse-develop/ passing we'll keep phoenix up (that poor old CentOS 6 box) for another month or so and then kill it. Or rather, plan its retirement party. 🎉 

In practice when things go south with phoenix because the server is down or whatever, I just click "Build Now" again after I've gotten phoenix back up. I don't sweat too much if the "develop" branch has advanced a bit. So I don't think we should sweat it too much here. Maybe the answer is that I (or anyone) should just click "Build Now" at https://jenkins.dataverse.org/job/IQSS-dataverse-develop/ if there's an EC2-related failure? Again, I'm just thinking this is something we could document in the dev guide.

**Comment by pdurbin** [comment]
I just tried clicking "Rerun" at https://jenkins.dataverse.org/blue/organizations/jenkins/IQSS-Dataverse-Develop-PR/detail/PR-9004/1/pipeline/

![Screen Shot 2022-09-29 at 3 19 44 PM](https://user-images.githubusercontent.com/21006/193123391-c76df872-308d-4150-815d-5a3ef6241c7e.png)

If this works, I'll probably just close this issue. It's intuitive enough. I got to the URL above from the pull request. I had to log in to click "Rerun" of course.
